DataType/NumberDataType.php

```diff
@@ -21,6 +21,7 @@ public function getAvailableFilterMethods()
             FilterCondition::METHOD_NUMERIC_LT,
             FilterCondition::METHOD_NUMERIC_NEQ,
             FilterCondition::METHOD_IN,
+            FilterCondition::METHOD_NIN,
```

Comment:
"NIN" because I saw Doctrine uses "NIN" instead of "NOT_IN".
But we could use "NOT_IN".